package.json

```diff
@@ -30,11 +30,15 @@
     "@babel/preset-env": "^7.4.5",
     "eslint": "^5.16.0",
     "husky": "^2.3.0",
-    "nodemon": "^1.19.0"
+    "nodemon": "^1.19.0",
+    "react-test-renderer": "^16.8.6"
```

Comment:
Remove this from package.json from this file because it has alredy been captured in button component